### PR TITLE
added close meeting functionality and change Email template

### PR DESCRIPTION
**Pull request by lbhssandilya**
added close meeting functionality and change Email template

### CODE REVIEW

File: ManageATenancyAPI/Services/Email/SendTraConfirmationEmailGateway.cs [hunk 1]
```diff
@@ -33,19 +33,29 @@ public async Task<SendTraConfirmationEmailOutputModel> SendTraConfirmationEmailA
             var tra = await _traAction.GetAsync(inputModel.TraId).ConfigureAwait(false);
             if(tra == null || tra.Email.IsNullOrEmpty())
                 return new SendTraConfirmationEmailOutputModel {IsSent = false,};
-
+            //change email subject based on sign off condition , if meeting is already signed off then sunject should be confirmation else for sign off
```
Comment (LBHMKeyworth) [inline ManageATenancyAPI/Services/Email/SendTraConfirmationEmailGateway.cs]: tiny typo in comments "sunject"

File: ManageATenancyAPI/Services/Email/SendTraConfirmationEmailGateway.cs [hunk 2]
```diff
@@ -33,19 +33,29 @@ public async Task<SendTraConfirmationEmailOutputModel> SendTraConfirmationEmailA
...
                 {EmailKeys.OfficerName, $"{inputModel.OfficerName}"},
                 {EmailKeys.OfficerAddress, $"{inputModel.OfficerAddress}"}
             };
 
-            await _client.SendEmailAsync(tra.Email, _config?.Value.TemplateId, personalization)
-                .ConfigureAwait(false);
+            //if the meeting is Signed off then send email with confirmation template else the template to sign Off the meeting
+            if (inputModel.IsMeetingSignedOff)
+            {
+                await _client.SendEmailAsync(tra.Email, _config?.Value.ConfirmationTemplate, personalization)
```
Comment (LBHMKeyworth) [inline ManageATenancyAPI/Services/Email/SendTraConfirmationEmailGateway.cs]: Are we able to deal with the emails not sending successfully or is that not a concern?

File: ManageATenancyAPI/UseCases/Meeting/SaveMeeting/SaveEtraMeetingUseCase.cs [hunk 3]
```diff
@@ -68,19 +71,36 @@ public async Task<SaveEtraMeetingOutputModel> ExecuteAsync(SaveETRAMeetingInputM
...
             {
-                MeetingId = meetingId,
+                MeetingId = response.InteractionId,
                 OfficerName = claims?.FullName,
-                TraId = request.TRAId
+                TraId = request.TRAId,
+                 IsMeetingSignedOff=outputModel.IsSignedOff
             };
 
+            //put this email in the else condition of Is Signed off
```
Comment (LBHMKeyworth) [inline ManageATenancyAPI/UseCases/Meeting/SaveMeeting/SaveEtraMeetingUseCase.cs]: Is this something that's been missed? - comment seems to suggest that the code needs to be moved? Or i'm misunderstanding :)

Comment (lbhssandilya) [reply ManageATenancyAPI/UseCases/Meeting/SaveMeeting/SaveEtraMeetingUseCase.cs]: I had added this comment just to remind me of the functionality ,that needed to be implemented ,
I will remove this

File: ManageATenancyAPI/Services/Email/SendTraConfirmationEmailGateway.cs (hunk 1 above)
Comment (lbhssandilya) [reply ManageATenancyAPI/Services/Email/SendTraConfirmationEmailGateway.cs]: Thanks for pointing out. I will correct It

File: ManageATenancyAPI/Services/Email/SendTraConfirmationEmailGateway.cs (hunk 2 above)
Comment (lbhssandilya) [reply ManageATenancyAPI/Services/Email/SendTraConfirmationEmailGateway.cs]: Gov.Notify takes care of sent/unsent emails and we refer  Gov.Notify for info

**Pull request merged**